src/camera.rs

```diff
@@ -156,12 +156,14 @@ impl OrbitControls {
     }
 
     pub fn update(&mut self, input: &Input) {
-        if !self.button.is_hit(input) {
+        if !self.button.is_hit(input) && !(input.get_mouse_wheel_movements().len()!=0) {
```

Comment:
You can check for `input.get_mouse_wheel() != 0`, but it's just a tip.
And please, add spaces around operators to make code more readable (like `.len() != 0` instead of `.len()!=0`